app/user/rest.py

```diff
@@ -53,7 +56,7 @@ def create_user():
 
 @user.route('/<uuid:user_id>', methods=['PUT'])
 def update_user(user_id):
```

Comment:
yep, plan is to remove this completely after this change and the password change is in.